test/common/index.js

```diff
@@ -139,19 +139,22 @@ if (process.env.NODE_TEST_WITH_ASYNC_HOOKS) {
       process._rawDebug();
       throw new Error(`same id added to destroy list twice (${id})`);
     }
-    destroyListList[id] = new Error().stack;
+    destroyListList[id] = util.inspect(new Error());
```

Comment:
What's an example of a situation where this will result in a better stack?

Reply:
Node.js core frames are visualized in grey. It makes it easier to distinguish the test code from core code.